clients/spark/core/src/main/scala/io/treeverse/clients/ApiClient.scala

```diff
@@ -57,7 +57,8 @@ class ApiClient(
     apiUrl: String,
     accessKey: String,
     secretKey: String,
-    connectionTimeout: String = ""
+    connectionTimeout: String = "",
+    readTimeout: String = "",
```

Comment:
There are so many params now! Can we just put them in a params static subclass? That way they will get names.

Reply:
opened https://github.com/treeverse/lakeFS/issues/3989 to follow up on these great comments